### ISSUE TITLE
Libraries error

### ISSUE THREAD

**Issue by GraphIgor** [IgorTimofeev/MineOS#310]
I tried to use some libraries, and i found not working, i tried to use this -->
local component = require("component")
local rs = component.redstone

rs.setOutput(require("sides").front, 15)

The library "sides" not found

**Comment by IgorTimofeev** [comment]
MineOS doesn't have sides library. If you need it - copy/paste from here:
https://github.com/MightyPirates/OpenComputers/blob/master-MC1.12/src/main/resources/assets/opencomputers/loot/openos/lib/sides.lua

**Comment by CoolCat467** [comment]
Actually... it does? It seems to be an older version of the one OpenOS has, but without some of the fancy features like iterating though the sides and whatnot. I did notice that "unknown" side was missing though.

What the real problem is, though, is that it's not referenced in the installer's file list, and it's not downloadable on the app market from what I saw the other day. At least it should be added to the app market, but it would probably just be easier to just have it added to that file list.

https://github.com/IgorTimofeev/MineOS/blob/master/Libraries/Sides.lua
